core/src/Core.php

```diff
@@ -432,11 +432,7 @@ public function sendRedirect($url, $count_attempts = 0, $type = 'REDIRECT_HEADER
                 );
                 exit;
             }
-            $url .= sprintf(
-                '%serr=%s',
-                Str::contains($url, '?') ? '?' : '&',
-                ($currentNumberOfRedirects + 1)
-            );
+            $url .= (Str::contains($url, '?') ? '?' : '&') . 'err=' . ($currentNumberOfRedirects + 1);
```

Comment:
The problem is not in lines :smile_cat: If Str::contains($url, '?') is true then & should be added, not ?